examples/simple/library.cc

```diff
@@ -0,0 +1,30 @@
+#include "opentelemetry/trace/provider.h"
+
+namespace trace = opentelemetry::trace;
+namespace nostd = opentelemetry::nostd;
+
+static nostd::shared_ptr<trace::Tracer> get_tracer()
+{
+  auto provider = trace::Provider::GetTracerProvider();
+  return provider->GetTracer("");
+}
+
+static void f1()
+{
+  auto span = get_tracer()->StartSpan("f1");
+}
```

Comment:
Please add some events. :)
Maybe in a followup PR?

Reply:
I'll add this in a follow-up PR. Events aren't wired up in the SDK yet.